pkg/api/client.go

```diff
@@ -1420,7 +1420,7 @@ type CreateProcessorPayload struct {
 	ClusterName string `json:"cluster,omitempty" yaml:"cluster"`
 	Namespace   string `json:"namespace,omitempty" yaml:"namespace"`
 	Pipeline    string `json:"pipeline" yaml:"pipeline"`     // defaults to Name if not set.
-	AppID       string `json:"appId,omitempty" yaml:"appId"` //not required
+	ProcessorID string `json:"appId,omitempty" yaml:"appId"` //not required
```

Comment:
the json tag remains the same though?

Reply:
This is deliberate, as the API has been designed to retain the old field name.

Reply:
So we update the field name only for the internal representation, however it goes through to the service via json as before